snuba/migrations/operations.py

```diff
@@ -49,12 +53,13 @@ def execute(self) -> None:
             nodes = dist_nodes
         else:
             raise ValueError(f"Target not set for {self}")
-
+        if nodes:
+            logger.info(f"Executing op: {self.format_sql()[:32]}...")
         for node in nodes:
             connection = cluster.get_node_connection(
                 ClickhouseClientSettings.MIGRATE, node
             )
-
+            logger.info(f"Executing on node: {node}")
```

Comment:
@dbanda do you think it would be worth adding whether it's a local or dist node here as well? i.e `Executing on local node:...`

Reply:
added